chapters/equations.tex

```diff
@@ -858,6 +860,39 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \end{lstlisting}
 \end{example}
 
+\begin{example}
+Resettable continuous-time controller initialized either in steady-state or by providing a \lstinline!start! value for state \lstinline!y!:
+\begin{lstlisting}[language=modelica]
+  parameter Boolean steadyState = true;
+  parameter Real y0 = 0 "start and reset value for y, if not steadyState";
+  Real y;
+equation
+  der(y) = a * y + b * u;
+  when {initial(), reset} then
```

Comment:
It would be nice to define 'reset'. Even in all State Machines examples it is usually defined or extended from somewhere where it is defined. Here without the context of State Machine it is not clear what it is supposed to be at all.

Reply:
Ok, I added a declaration of reset. I made it a non-connector input to avoid depending on MSL.